routes/admins.js

```diff
@@ -55,9 +56,13 @@ router.post('/', async (req, res) => {
         admin.rows[0].user_id,
         firebaseId,
         'admin',
-        'pending', // admin status is pending until email is verified
+        'active', // verified by default since it was through invite
```

Comment:
Does this mean that we no longer use `pending` at all? If so, we may need to consider removing it from the schema and making active true/false again.

Reply:
We possibly don't anymore. Instead of the account being created once the admin invites the user (which would lead to a pending status), the account is created after the invite process goes through (edge case with invite expiring and there is a firebase account already created so more work for us to check if there is an existing account to resend an invite). So, we would not need the pending status, so we could change back it back being a boolean. Do we necessarily need a pending status?